src/bin/cargo-miri.rs

```diff
@@ -167,20 +167,22 @@ fn ask(question: &str) {
     io::stdout().flush().unwrap();
     io::stdin().read_line(&mut buf).unwrap();
     match buf.trim().to_lowercase().as_ref() {
-        "" | "y" | "yes" => {}, // proceed
+        "" | "y" | "yes" => (), // proceed
```

Comment:
The [style guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/expressions.md) says

> An empty block should be written as `{}`.

Reply:
Aha, didn't know that, thanks.